docs/guide/arktos_mizar_installation_single.md

```diff
@@ -0,0 +1,140 @@
+<!--
+SPDX-License-Identifier: MIT
+Copyright (c) 2020 The Authors.
+
+Authors: Catherine Lu      <@clu2>
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:The above copyright
+notice and this permission notice shall be included in all copies or
+substantial portions of the Software.THE SOFTWARE IS PROVIDED "AS IS",
+WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
+FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR
+THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+-->
+
+# Arktos and Mizar Single Node Installation Guide
+
+## Introduction
+
+This document is intended for new users to install Arktos platform with Mizar as the underlying network technology. 
+
+For more details on Arktos installation, please refer to [this link](https://github.com/centaurus-cloud/arktos/blob/master/docs/setup-guide/arktos-enforces-network-feature.md)
+
+## Installation Steps
+1. Prepare lab machine, the prefered OS is **Ubuntu 18.04**. If you are using AWS, the recommanded instance size is ```t2.2xlarge``` and the storage size is ```128GB``` or more 
+
+```bash
+wget https://raw.githubusercontent.com/Hong-Chang/tools/master/setup-machine-arktos.sh && chmod 755 setup-machine-arktos.sh && ./setup-machine-arktos.sh
+
+```
+Once above script is completed, you will be automatically logged out of the lab machine. 
+
+2. You will need ssh into the same lab machine again, and download mizar source code onto your lab machine: 
+
+```bash
+cd /home/ubuntu/go/src/k8s.io/
+git clone https://github.com/centaurus-cloud/mizar.git

```

Comment:
Since the script has cloned mizar under ~/mizar, we can just use that.
It's not proper to put mizar repo under k8s.io. The arktos repo is under k8s.io because there are some limitation leading to better to put arktos under k8s.io.